examples/llama_index_multimodal_rag.ipynb

```diff
@@ -0,0 +1,332 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b1acc12a-712e-41e6-8e30-41f9de223543",
+   "metadata": {},
+   "source": [
+    "# Multimodal RAG with LlamaIndex"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4c557723-257f-4746-9b84-ec77c50cf405",
+   "metadata": {},
+   "source": [
+    "This cookbook shows how to perform RAG on the table and text extraction output of nv-ingest's pdf extraction tools using LlamaIndex"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "baecfda5-137b-43da-a8d4-23dd47131be9",
+   "metadata": {},
+   "source": [
+    "To start we'll need to make sure we have llama_index installed"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Similar to langchain mention or reference how to spin up a local nv-ingest instance so this example works.